pkg/scheduler/metrics/metrics.go

```diff
@@ -301,6 +308,38 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"type"})
 
+	ReadyTime = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Subsystem: SchedulerSubsystem,
+			Name:      "ready_time",
+			Help:      "Ready time for leader-scheduler to work.",
+		}, []string{"hostname"},
+	)
+
+	StartLatency = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Subsystem: SchedulerSubsystem,
+			Name:      "start_latency",
+			Help:      "Latency in microseconds for leader-scheduler to be ready after init.",
+		}, []string{"hostname"},
+	)
+
+	InitLatency = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Subsystem: SchedulerSubsystem,
+			Name:      "init_latency",
+			Help:      "Latency in microseconds for scheduler to be pod cache populated.",
```

Comment:
The `init_latency` and `start_latency` could be confusing. This one seems to be the `cache_init_latency`. Can we rename it and make it clear in the `Help` text?

Reply:
rename as `cache_init_latency_seconds` and `leader_ready_latency_seconds`